frontend/js/recordings.js

```diff
@@ -146,3 +170,27 @@ function mapWordformsToBestRecordingURL(response) {
 
   return wordform2recordingURL;
 }
+
+/**
+ * Chunks the iterable into subarrays of a maximum size.
+
+ * @param {Iterable<T>}  one or more wordforms to search for.
+ * @return {Array<Array<T>}}
+ */
+function chunk(collection) {
+  const MAX_BATCH_SIZE = 50;
+
+  let chunks = [[]];
+  for (let item of collection) {
+    let currentChunk = chunks[chunks.length - 1];
```

Comment:
Why not always use `-1` if you're looking for the last item in the array?
```suggestion
    let currentChunk = chunks[-1];
```

Reply:
Unfortunatly JavaScript does not work like that :(

(I really wish it did).

Reply:
JavaScript is in the _Cobra, D, Haxe, Java, Scala_ language of familys where the most concise built-in way to get the last item is `array[array.length - 1]`. See: https://en.wikipedia.org/wiki/Comparison_of_programming_languages_(array)#Array_dimensions

Reply:
Ah, in my mind it was already doing that at least once. I thought `[[]]` had length 0, not length 1 for some reason